### PR TITLE
changed trial period to 90 days

### PR DESCRIPTION
**Pull request by MelanieCueppers**

### CODE REVIEW

File: tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md [hunk 1]
```diff
@@ -1,138 +1,126 @@
...
 
-So you've signed up for SAP BTP trial but are not quite sure where to start? This tutorial is here to help!
+So, you've signed up for SAP BTP trial but are not quite sure where to start? This tutorial is here to help!
 
-A trial account on SAP Business Technology Platform enables you to experience it for free, offering access to a comprehensive set of platform services, as well as early access to beta functionality.
+A trial account on SAP Business Technology Platform enables you to experience the platform for free for a limited amount of time, offering access to a comprehensive set of platform services, as well as early access to beta functionality. Trial accounts are intended for personal exploration, and not for production use or team development. They allow restricted use of the platform resources and services.
 
-Trial accounts are intended for personal exploration, and not for production use or team development. They allow restricted use of the platform resources and services.
+**Note:** You want to explore SAP Business Technology Platform without time constraints? See [Get an Account on SAP BTP to Try Out Free Tier Service Plans](btp-free-tier-account)
```
Comment (MichaelCzcz) [inline tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md:33]: Please remove the label "note" or replace it with a different one. The formatting already makes it a note.

File: tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md [hunk 2]
```diff
@@ -1,138 +1,126 @@
...
 
-You can use your SAP BTP trial account for up to 365 days in total. However, your trial account is suspended after 30 days but don't worry, you can easily extend that time by clicking **Extend Trial** in the popup window that appears once your trial account has been suspended. After 365 days total, you trial account is deleted but you can always create a new account. For more information, see [Trial Accounts](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/046f127f2a614438b616ccfc575fdb16.html).
+You can use your SAP BTP trial account for up to 90 days in total. However, your trial account is suspended after 30 days but don't worry, you can easily extend that time by selecting **Extend Trial** in the popup window that appears once your trial account has been suspended. After 90 days total, you trial account is deleted but you can always create a new account. For more information, see [Trial Accounts](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/046f127f2a614438b616ccfc575fdb16.html).
```
Comment (MichaelCzcz) [inline tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md:35]: please fix misspelling/typo

File: tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md [hunk 3]
```diff
@@ -1,138 +1,126 @@
...
-There are 2 ways of navigating in the cockpit:
+There are two ways of navigating in the cockpit:
 
-- using the left hand-side navigation
-- using the breadcrumbs at the top of the page
+- using the left hand-side navigation,
+- or using the breadcrumbs at the top of the page.
 
-There are certain actions you can perform at each hierarchical level (global account, subaccount, space etc.) - we will refer to the sum of these actions as a **scope**. For example, when you are in a global account you can add global account members, create subaccounts and add entitlements to subaccounts. They represent the **global account scope**.
+There are certain actions you can perform at each hierarchical level (global account, subaccount, space etc.). We will refer to the sum of these actions as a **scope**. For example, when you are in a global account, you can add global account members, create subaccounts, and add entitlements to subaccounts. They represent the **global account scope**.
```
Comment (MichaelCzcz) [inline tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md:109]: Don't use "we" unless you are referring to SAP.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by MichaelCzcz** [comment]
Please resolve conflicts. Always first pull your fork before you push any changes to production!

File: tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md (hunk 1 above)
Comment (MelanieCueppers) [reply tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md]: Done

File: tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md (hunk 3 above)
Comment (MelanieCueppers) [reply tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md]: fixed

**Review by MelanieCueppers — COMMENTED** [review]
All changes done. Please merge pull request. :)

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
image
Correct tags and add missing comma before "but" in line 35.

**Comment by MelanieCueppers** [comment]
tags corrected and comma added

**Pull request merged**